relay.h

```diff
@@ -22,7 +22,7 @@
 #include "config.h"
 #endif
 
-#define METRIC_BUFSIZ 8192
+#define METRIC_BUFSIZ 32768
```

Comment:
I think this is ok, it should only add some memory usage on a constant factor.  It always is used with stack allocated memory.